src/core/mac/mac.cpp

```diff
@@ -748,7 +768,9 @@ void Mac::HandleBeginTransmit(void)
 
 extern "C" void otPlatRadioTransmitDone(otInstance *aInstance, bool aRxPending, ThreadError aError)
 {
+    otLogFuncEntryMsg("%!otError!, aRxPending=%u", aError, aRxPending ? 1 : 0);
```

Comment:
Ah, yes. It is Windows specific. It automatically converts enums. The easiest way to handle this is to add a `#ifdef WINDOWS_LOGGING` here and change the format string. Would that be ok in this case?